src/helpers/message.ts

```diff
@@ -157,25 +191,55 @@ export function newOrderMessage(order: OrderDto, baseUrl: string): string {
   )
 
   // unlimited?
-  const isUnlimited = isOrderUnlimited(priceNumerator, priceDenominator)
 
   // TODO: Should we publish even if the user doesn't have balance. Should we include the balance of the user? he can change it...
   //  https://github.com/gnosis/dex-telegram/issues/45
 
-  const sellMsg = buildSellMsg(isUnlimited, buyTokenLabel, sellTokenLabel, buyAmountFmt, sellAmountFmt)
-  // Calculate the price
+  // Partial message: Order description
+  //    i.e Sell 3.535 WETH for 1,000 SNX
+  const isUnlimited = isOrderUnlimited(priceNumerator, priceDenominator)
+  const sellMsg = buildSellMsg(isUnlimited, buyTokenLabel, sellTokenLabel, buyAmountFmt, sellAmountFmt) + '\n'
```

Comment:
Would be nice to have object as params for `buildSellMsg`

Reply:
40457d71e83aa649b8ebdf7e90b231ac692e1a49